app/models/katello/glue/candlepin/pool.rb

```diff
@@ -115,11 +115,11 @@ def import_data(index_hosts = true)
         pool_attributes[:pool_type] = pool_json["type"] if pool_json.key?("type")
 
         if pool_attributes.key?(:multi_entitlement)
-          pool_attributes[:multi_entitlement] = pool_attributes[:multi_entitlement] == "yes" ? true : false
+          pool_attributes[:multi_entitlement] = pool_attributes[:multi_entitlement]
         end
 
         if pool_attributes.key?(:virtual)
-          pool_attributes[:virt_only] = pool_attributes["virtual"] == 'true' ? true : false
```

Comment:
shouldn't this still have `== 'true'` on the end?

Reply:
yes, I had that changed back locally and guess I didn't push up!